RecoLocalTracker/SiStripClusterizer/plugins/ClustersFromRawProducer.cc

```diff
@@ -444,7 +443,9 @@ void ClusterFiller::fill(StripClusterizerAlgorithm::output_t::FastFiller & recor
   if(!record.empty()) incNoZ();
 
   // COUT << "filled " << record.size() << std::endl;
-  
+} catch (edmNew::CapacityExaustedException) {
```

Comment:
No problem
now is cough at line  348!
(
the only one that counts is
READOUT_MODE_ZERO_SUPPRESSED_LITE
)
and yes most probably
should catch before
```
clusterizer.stripByStripEnd(record);
  incAct();
  if(!record.empty()) incNoZ();
```